### PR TITLE
Adds whitelist annotation support to CC

### PR DESCRIPTION
**Pull request by caphrim007**
This patch adds whitelist annotation processing to CC. Whitelist
entries are created as part of L7 policies.

### CODE REVIEW

File: pkg/appmanager/routing.go [hunk 1]
```diff
@@ -19,17 +19,16 @@ package appmanager
...
 	"sort"
 	"strconv"
 	"strings"
-	"sync"
-
-	log "github.com/F5Networks/k8s-bigip-ctlr/pkg/vlogger"
 
 	routeapi "github.com/openshift/origin/pkg/route/api"
 	"k8s.io/client-go/pkg/apis/extensions/v1beta1"
+	"sync"
```
Comment (sjberman) [inline pkg/appmanager/routing.go]: Leave the import statements as they were. Standard go convention for import structure is:

1. standard library
2. third party
3. internal packages

**Comment by sjberman** [comment]
Also, please add this feature to the release notes for the next release.

If you could run the functional tests to verify nothing else was broken, that would be great.

**Comment by caphrim007** [comment]
@sjberman ready for review.

**Comment by sjberman** [comment]
Could you add a small test case(s) to the "merges and sorts rules correctly with matching conditions" unit test in resourceConfig_test.go to verify merging/unmerging of this new rule with existing rules works properly?

**Comment by caphrim007** [comment]
@sjberman ready for review

**Pull request merged**